cli-feed-v4.json

```diff
@@ -46,9 +46,9 @@
       "hidden": false
     },
     "v4-prerelease": {
-      "release": "4.83.0",
+      "release": "4.84.0",
       "releaseQuality": "Prerelease",
-      "hidden": true
+      "hidden": false
```

Comment:
The hidden flag should not change for the tag.

Reply:
My bad. I was just using it to test locally. Resolved it.